### ISSUE TITLE
Unexpected behaviour of repo argument with other values than url

### ISSUE THREAD

**Issue by asdf8601** [profclems/glab#589]
**Description**

Problems with the values allowed by `repo` argument.

* glab get stucked when I execute the below code from other repository, it doesn't fail but nothing happens. 

```
glab issue list -R GROUP/SUBGROUP/PROJECT 
```

* glab fails when I use a project id

```
glab issue list -R PROJECT_ID
```

However, using the url works well.

**Expected Behavior vs Actual Behavior**

**Possible Fix**

**Steps to Reproduce**

1. Type this '...'
2. View the output '....'
3. See error

**Logs**

```
~/gitlab/grp/libraries/my_prj issues/237-t..n-estructure *3 !1 ?1                                            py grp todo 3 08:16:18
> glab issue list --repo 324
expected the "[HOST/]OWNER/[NAMESPACE/]REPO" format, got "324"

~/gitlab/grp/libraries/grp issues/237-t..n-estructure *3 !1 ?1                                            py grp todo 3 08:16:30
> glab issue list --repo GROUP/SUBGRP/prj
^C
~/gitlab/grp/libraries/grp issues/237-t..n-estructure *3 !1 ?1                                    INT 15s py grp todo 3 08:17:02
> glab issue list --repo GROUP/SUBGRP/prj

^C
~/gitlab/grp/libraries/grp issues/237-t..n-estructure *3 !1 ?1                                    INT 26s py grp todo 3 08:17:39
>

```
**Your Environment**

* Version used (Run `glab --version`): glab version 1.13.1 (2020-12-28)
* Operating System and version: Ubuntu 20.04.1 LTS

**Comment by profclems** [comment]
Thanks @mmngreco for reporting this issue!

I'm well aware of this issue. I will add a fix and release as part of v1.14.0 today

**Comment by asdf8601** [comment]
Awesome, thank you for your support! ;-)
